app/assets/stylesheets/frontend/print/_html-publication.scss

```diff
@@ -0,0 +1,21 @@
+.html-publications-show {
+  nav.in-page-navigation {
+    display: block!important;
```

Comment:
Is there a way to achieve this without an `important`? Using `important` is a last resort we should really avoid.